frontend/components/Course/Roster/Roster.tsx

```diff
@@ -464,6 +471,49 @@ const Roster = (props: RosterProps) => {
                                     </Table.Row>
                                 ))}
                             </Table.Body>
+                            <Table.Footer>
+                                {memberships && (
+                                    <Table.Row textAlign="right">
+                                        <Table.HeaderCell colSpan="6">
+                                            <Pagination
+                                                activePage={activePageState}
+                                                totalPages={Math.max(
+                                                    1,
+                                                    Math.ceil(
+                                                        filteredUsers.length /
+                                                            MAX_MEMBERS_PER_PAGE
+                                                    )
+                                                )}
+                                                onPageChange={(
```

Comment:
Can you clean up this function and also update the summary table with the same? There really should be a one or two line equivalent to this.

Reply:
At the very least if you set activePage's type to number | string | undefined you can just set it directly. Summary might be a bit weird though but take a look at that well.

Reply:
Refactored with fancy ternary function. Must define new variable as linter isn't happy if we redefine function parameter.